code/datums/emergency_calls/skeleton.dm

```diff
@@ -0,0 +1,40 @@
+/datum/emergency_call/skeleton
+	name = "The Bone Zone"
+	probability = 0
+	spawn_type = /mob/living/carbon/human/species/skeleton
+
+/datum/emergency_call/skeleton/print_backstory(mob/living/carbon/human/H)
+	to_chat(H, "<B>You are a skeleton, risen from beyond the grave.")
+	to_chat(H, "<B>Long have you rested, ejected into the void of space through battle or unfortunate circumstances, but now you have awoken.</B>")
+	to_chat(H, "<B>The distress call has awoken you from your spacebound slumber.</b>")
+	to_chat(H, "<B>Wipe out whatever has disturbed your eternal rest, so you may sleep once again.</b>")
```

Comment:
use \n to rill these into one tochat

Reply:
![image](https://user-images.githubusercontent.com/44979174/96272328-dc10e200-0fc5-11eb-9ab1-4eed6e9e47a5.png)

I'm mad because none of the other ERTs use \n